kolibri/core/auth/test/test_morango_integration.py

```diff
@@ -32,8 +32,10 @@
 from kolibri.core.lessons.models import LessonAssignment
 from kolibri.core.logger.models import ContentSessionLog
 from kolibri.core.logger.models import ContentSummaryLog
-from kolibri.core.logger.models import ExamAttemptLog
-from kolibri.core.logger.models import ExamLog
+
+# TODO: add back in after single-user exam assignment is reinstated
+# from kolibri.core.logger.models import ExamAttemptLog
+# from kolibri.core.logger.models import ExamLog
```

Comment:
Because they won't pass until we reinstate single-user quiz syncing, I commented out the related test portions here (with a "TODO" by each, cited in the commented-out plugin file).